indexers/vector/FaissIndexer/tests/test_faissindexer.py

```diff
@@ -211,3 +211,23 @@ def test_indexer_train_from_index_different_compression_levels(metas, compressio
         idx, dist = indexer.query(query, top_k=4)
         assert idx.shape == dist.shape
         assert idx.shape == (num_query, 4)
+
+def test_faiss_normalization(metas):
+    num_data = 1
+    num_dims = 64
+    vec_idx = np.random.randint(0, high=num_data, size=[num_data])
+    with FaissIndexer(index_key='Flat', distance="inner_product", normalize=True) as indexer:
+        vecs = np.zeros((num_data, num_dims))
+        vecs[0, 0] = 1
+        indexer.add(vec_idx, vecs)
+        indexer.save()
+        save_abspath = indexer.save_abspath
+
+
+    with BaseIndexer.load(save_abspath) as indexer:
+        query = np.zeros((1, num_dims))
+        query[0, 0] = 1
```

Comment:
can u have query with a value of 5? that would be a better proof of normalization?

Reply:
and shouldn't distance be 0?

Reply:
I'll do that. The cosine similarity ranges from -1 to 1, where 1 is perfectly similar. I added it to the documentation. For l2 0 means perfectly similar.

Reply:
oh thanks, I thought it would compute the distance and not the similarity. Could you add a test with more indexed documents (to show the order in which the documents are returned?)

Thanks

Reply:
@JoanFM Sure, but there is a problem when adding multiple vectors, the index acts very weird.
Most of the time the index is correct. But sometimes two vectors have the same index.
Result of my query:
```
[[1 0]] [[1.        0.5547002]]
```

Reply:
@JoanFM I probably add the vectors in the wrong way:
```
  vecs = np.zeros((num_data, num_dims))
  vecs[:, 0] = 2
  vecs[0, 1] = 3
  keys = np.array([0, 1]).reshape(-1, 1)
  indexer.add(keys, vecs)
```

Reply:
will try to look at it

Reply:
so did u figure it out?